include/config/dexnav.h

```diff
@@ -0,0 +1,65 @@
+#ifndef GUARD_CONFIG_DEXNAV_H
+#define GUARD_CONFIG_DEXNAV_H
+
+
+#define USE_DEXNAV_SEARCH_LEVELS      FALSE  /* WARNING: EXCEEDS SAVEBLOCK SPACE! YOU WILL NEED TO FREE UP SPACE */
```

Comment:
there space in saveblock 3 so the warning is not accurate.

How much space is actually needed?